dns/dns_updater.go

```diff
@@ -0,0 +1,97 @@
+package dns
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/sky-uk/feed/controller"
+	"github.com/sky-uk/feed/dns/r53"
+	"github.com/sky-uk/feed/elb"
+	"k8s.io/heapster/Godeps/_workspace/src/github.com/coreos/fleet/log"
+)
+
+type updater struct {
+	dns          r53.Route53Client
+	elb          elb.ELB
+	elbLabelName string
+}
+
+// New creates an updater for dns
+func New(dnsClient r53.Route53Client, awsElb elb.ELB, elbLabelName string) controller.Updater {
```

Comment:
Any reason we don't create the route53 client internally here? This is bleeding the aws dependency out of the dns package.